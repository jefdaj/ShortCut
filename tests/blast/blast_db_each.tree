.
|-- cache
|   |-- each
|   |   |-- 0390136808
|   |   |   |-- 67ef4ead59.bht -> ../../../exprs/blastn_db/ce1c174684_3d999d2434_67ef4ead59.bht
|   |   |   |-- 67ef4ead59.bht.args -> ../../../cache/lines/...
|   |   |   |-- 6b02a1f1da.bht -> ../../../exprs/blastn_db/ce1c174684_3d999d2434_6b02a1f1da.bht
|   |   |   `-- 6b02a1f1da.bht.args -> ../../../cache/lines/...
|   |   |-- 0665aa6554
|   |   |   |-- 5d49ce52c6.str.list -> ../../../exprs/extract_queries/5d49ce52c6.str.list
|   |   |   |-- 5d49ce52c6.str.list.args -> ../../../cache/lines/...
|   |   |   |-- 934a55faaa.str.list -> ../../../exprs/extract_queries/934a55faaa.str.list
|   |   |   `-- 934a55faaa.str.list.args -> ../../../cache/lines/...
|   |   |-- 09160f2aec
|   |   |   |-- 5d49ce52c6.str.list -> ../../../exprs/extract_queries/5d49ce52c6.str.list
|   |   |   |-- 5d49ce52c6.str.list.args -> ../../../cache/lines/...
|   |   |   |-- 934a55faaa.str.list -> ../../../exprs/extract_queries/934a55faaa.str.list
|   |   |   `-- 934a55faaa.str.list.args -> ../../../cache/lines/...
|   |   |-- 11cc1d0ab4
|   |   |   |-- 15416686ae.str.list -> ../../../exprs/extract_queries/15416686ae.str.list
|   |   |   |-- 15416686ae.str.list.args -> ../../../cache/lines/...
|   |   |   |-- 2445e66ccd.str.list -> ../../../exprs/extract_queries/2445e66ccd.str.list
|   |   |   `-- 2445e66ccd.str.list.args -> ../../../cache/lines/...
|   |   |-- 28e4a6b4de
|   |   |   |-- 15416686ae.str.list -> ../../../exprs/extract_queries/15416686ae.str.list
|   |   |   |-- 15416686ae.str.list.args -> ../../../cache/lines/...
|   |   |   |-- 2445e66ccd.str.list -> ../../../exprs/extract_queries/2445e66ccd.str.list
|   |   |   `-- 2445e66ccd.str.list.args -> ../../../cache/lines/...
|   |   |-- 3219e28c5c
|   |   |   |-- 2c5c740444.bht -> ../../../exprs/blastp_db/ce1c174684_6b304f493c_2c5c740444.bht
|   |   |   |-- 2c5c740444.bht.args -> ../../../cache/lines/...
|   |   |   |-- 58ea568aa7.bht -> ../../../exprs/blastp_db/ce1c174684_6b304f493c_58ea568aa7.bht
|   |   |   `-- 58ea568aa7.bht.args -> ../../../cache/lines/...
|   |   |-- 444fc67f9b
|   |   |   |-- 2c5c740444.bht -> ../../../exprs/blastp/ce1c174684_6b304f493c_2c5c740444.bht
|   |   |   |-- 2c5c740444.bht.args -> ../../../cache/lines/...
|   |   |   |-- 58ea568aa7.bht -> ../../../exprs/blastp/ce1c174684_6b304f493c_58ea568aa7.bht
|   |   |   `-- 58ea568aa7.bht.args -> ../../../cache/lines/...
|   |   |-- 514705c620
|   |   |   |-- 6af3464344.str.list -> ../../../exprs/extract_queries/6af3464344.str.list
|   |   |   |-- 6af3464344.str.list.args -> ../../../cache/lines/...
|   |   |   |-- a7b4e17972.str.list -> ../../../exprs/extract_queries/a7b4e17972.str.list
|   |   |   `-- a7b4e17972.str.list.args -> ../../../cache/lines/...
|   |   |-- 537694df95
|   |   |   |-- 67ef4ead59.bht -> ../../../exprs/blastn/ce1c174684_3d999d2434_67ef4ead59.bht
|   |   |   |-- 67ef4ead59.bht.args -> ../../../cache/lines/...
|   |   |   |-- 6b02a1f1da.bht -> ../../../exprs/blastn/ce1c174684_3d999d2434_6b02a1f1da.bht
|   |   |   `-- 6b02a1f1da.bht.args -> ../../../cache/lines/...
|   |   |-- 6ab9c310ea
|   |   |   |-- 299f46e311.str.list -> ../../../exprs/extract_queries/299f46e311.str.list
|   |   |   |-- 299f46e311.str.list.args -> ../../../cache/lines/...
|   |   |   |-- 4851018b18.str.list -> ../../../exprs/extract_queries/4851018b18.str.list
|   |   |   `-- 4851018b18.str.list.args -> ../../../cache/lines/...
|   |   |-- a150927fa3
|   |   |   |-- 23f474a06c.ndb -> ../../../exprs/makeblastdb_nucl/23f474a06c.ndb
|   |   |   |-- 23f474a06c.ndb.args -> ../../../cache/lines/...
|   |   |   |-- 947877ffa6.ndb -> ../../../exprs/makeblastdb_nucl/947877ffa6.ndb
|   |   |   `-- 947877ffa6.ndb.args -> ../../../cache/lines/...
|   |   |-- d8588bd737
|   |   |   |-- d0cc72f231.pdb -> ../../../exprs/makeblastdb_prot/d0cc72f231.pdb
|   |   |   |-- d0cc72f231.pdb.args -> ../../../cache/lines/...
|   |   |   |-- e51c741dc5.pdb -> ../../../exprs/makeblastdb_prot/e51c741dc5.pdb
|   |   |   `-- e51c741dc5.pdb.args -> ../../../cache/lines/...
|   |   |-- e4e4ba8f69
|   |   |   |-- ce492d3420.str.list -> ../../../exprs/extract_queries/ce492d3420.str.list
|   |   |   |-- ce492d3420.str.list.args -> ../../../cache/lines/...
|   |   |   |-- ee21534918.str.list -> ../../../exprs/extract_queries/ee21534918.str.list
|   |   |   `-- ee21534918.str.list.args -> ../../../cache/lines/...
|   |   `-- f3257b5bbe
|   |       |-- 36a5468151.str.list -> ../../../exprs/extract_queries/36a5468151.str.list
|   |       |-- 36a5468151.str.list.args -> ../../../cache/lines/...
|   |       |-- 9e08b54850.str.list -> ../../../exprs/extract_queries/9e08b54850.str.list
|   |       `-- 9e08b54850.str.list.args -> ../../../cache/lines/...
|   |-- load
|   |   |-- 0238b7dcea.fna
|   |   |-- 0238b7dcea.fna.ids
|   |   |-- 32841598b2.faa
|   |   |-- 32841598b2.faa.ids
|   |   |-- 737dfdb34e.fna
|   |   |-- 737dfdb34e.fna.ids
|   |   |-- a810086108.faa
|   |   |-- a810086108.faa.ids
|   |   |-- de2033f5c0.faa
|   |   |-- de2033f5c0.faa.ids
|   |   |-- ff08c7f09e.fna
|   |   `-- ff08c7f09e.fna.ids
|   `-- makeblastdb
|       |-- b025a78684
|       |   |-- b025a78684.pdb.phr
|       |   |-- b025a78684.pdb.pin
|       |   `-- b025a78684.pdb.psq
|       |-- d24c42156b
|       |   |-- d24c42156b.ndb.nhr
|       |   |-- d24c42156b.ndb.nin
|       |   `-- d24c42156b.ndb.nsq
|       |-- e00fb07ddb
|       |   |-- e00fb07ddb.pdb.phr
|       |   |-- e00fb07ddb.pdb.pin
|       |   `-- e00fb07ddb.pdb.psq
|       `-- e1e5fdbeba
|           |-- e1e5fdbeba.ndb.nhr
|           |-- e1e5fdbeba.ndb.nin
|           `-- e1e5fdbeba.ndb.nsq
|-- exprs
|   |-- any
|   |   |-- 0c93ce5921.str.list.list -> ../../cache/lines/...
|   |   |-- 3d347437fe.str.list -> ../../cache/lines/...
|   |   |-- 77b28c70bc.str.list.list -> ../../cache/lines/...
|   |   |-- b774e4798e.str.list.list -> ../../cache/lines/...
|   |   |-- ba83c277a2.str.list.list -> ../../cache/lines/...
|   |   |-- e178602817.str.list.list -> ../../cache/lines/...
|   |   |-- e2afcd9cdf.str.list.list -> ../../cache/lines/...
|   |   `-- eecb405c87.str.list.list -> ../../cache/lines/...
|   |-- blastn
|   |   |-- ce1c174684_3d999d2434_67ef4ead59.bht
|   |   `-- ce1c174684_3d999d2434_6b02a1f1da.bht
|   |-- blastn_db
|   |   |-- 8158675169_bc4ea1fc06_a78dfa6d35.bht
|   |   |-- 8158675169_bc4ea1fc06_d326247d21.bht
|   |   |-- ce1c174684_3d999d2434_67ef4ead59.bht
|   |   `-- ce1c174684_3d999d2434_6b02a1f1da.bht
|   |-- blastn_db_each
|   |   `-- 8158675169_bc4ea1fc06_992cadafb5.bht.list -> ../../cache/lines/...
|   |-- blastn_each
|   |   `-- 8158675169_bc4ea1fc06_43095607a3.bht.list -> ../../cache/lines/...
|   |-- blastp
|   |   |-- ce1c174684_6b304f493c_2c5c740444.bht
|   |   `-- ce1c174684_6b304f493c_58ea568aa7.bht
|   |-- blastp_db
|   |   |-- 8158675169_1af4d085e7_46b5852d89.bht
|   |   |-- 8158675169_1af4d085e7_64ccb38416.bht
|   |   |-- ce1c174684_6b304f493c_2c5c740444.bht
|   |   `-- ce1c174684_6b304f493c_58ea568aa7.bht
|   |-- blastp_db_each
|   |   `-- 8158675169_1af4d085e7_5d8e23195d.bht.list -> ../../cache/lines/...
|   |-- blastp_each
|   |   `-- 8158675169_1af4d085e7_cd45a3683c.bht.list -> ../../cache/lines/...
|   |-- extract_queries
|   |   |-- 15416686ae.str.list -> ../../cache/lines/...
|   |   |-- 2445e66ccd.str.list -> ../../cache/lines/...
|   |   |-- 299f46e311.str.list -> ../../cache/lines/...
|   |   |-- 36a5468151.str.list -> ../../cache/lines/...
|   |   |-- 4851018b18.str.list -> ../../cache/lines/...
|   |   |-- 5d49ce52c6.str.list -> ../../cache/lines/...
|   |   |-- 6af3464344.str.list -> ../../cache/lines/...
|   |   |-- 934a55faaa.str.list -> ../../cache/lines/...
|   |   |-- 9e08b54850.str.list -> ../../cache/lines/...
|   |   |-- a7b4e17972.str.list -> ../../cache/lines/...
|   |   |-- ce492d3420.str.list -> ../../cache/lines/...
|   |   `-- ee21534918.str.list -> ../../cache/lines/...
|   |-- extract_queries_each
|   |   |-- 2162b979c9.str.list.list -> ../../cache/lines/...
|   |   |-- 287949ec2e.str.list.list -> ../../cache/lines/...
|   |   |-- 562328bf5a.str.list.list -> ../../cache/lines/...
|   |   |-- a26d5c2d08.str.list.list -> ../../cache/lines/...
|   |   |-- a784742252.str.list.list -> ../../cache/lines/...
|   |   |-- ba3c5c7d5d.str.list.list -> ../../cache/lines/...
|   |   |-- c5313a01de.str.list.list -> ../../cache/lines/...
|   |   `-- d46672faa3.str.list.list -> ../../cache/lines/...
|   |-- list
|   |   |-- 03504a938c.str.list.list.list -> ../../cache/lines/...
|   |   |-- 0db8510f64.bht.list -> ../../cache/lines/...
|   |   |-- 1dd9b326bf.ndb.list -> ../../cache/lines/...
|   |   |-- 40351a2268.str.list.list.list -> ../../cache/lines/...
|   |   |-- 487f4dec22.bht.list -> ../../cache/lines/...
|   |   |-- 6e89b2d656.str.list.list.list -> ../../cache/lines/...
|   |   |-- 744b048cab.fna.list -> ../../cache/lines/...
|   |   |-- 80dd23d080.str.list.list.list -> ../../cache/lines/...
|   |   |-- 9055afcb7e.fna.list -> ../../cache/lines/...
|   |   |-- b94143526c.str.list.list.list -> ../../cache/lines/...
|   |   |-- bb839d1e83.faa.list -> ../../cache/lines/...
|   |   |-- cadff7be88.str.list.list.list -> ../../cache/lines/...
|   |   |-- cf3f423886.faa.list -> ../../cache/lines/...
|   |   |-- d56def961e.pdb.list -> ../../cache/lines/...
|   |   |-- fb4802f11e.fna.list -> ../../cache/lines/...
|   |   |-- fc371e783f.faa.list -> ../../cache/lines/...
|   |   `-- fd1b39d770.str.list.list.list -> ../../cache/lines/...
|   |-- load_faa
|   |   |-- 2d88028e40.faa -> ../../cache/load/32841598b2.faa
|   |   |-- 387f499ddc.faa -> ../../cache/load/a810086108.faa
|   |   `-- e910806f83.faa -> ../../cache/load/de2033f5c0.faa
|   |-- load_fna
|   |   |-- 82b637da24.fna -> ../../cache/load/0238b7dcea.fna
|   |   |-- eb12539f1c.fna -> ../../cache/load/ff08c7f09e.fna
|   |   `-- f110b4b1c9.fna -> ../../cache/load/737dfdb34e.fna
|   |-- makeblastdb_nucl
|   |   |-- 17e9ac9897.ndb -> ../../cache/lines/...
|   |   |-- 23f474a06c.ndb -> ../../cache/lines/...
|   |   |-- 611e5b4cfb.ndb -> ../../cache/lines/...
|   |   `-- 947877ffa6.ndb -> ../../cache/lines/...
|   |-- makeblastdb_nucl_each
|   |   `-- 1c52697303.ndb.list -> ../../cache/lines/...
|   |-- makeblastdb_prot
|   |   |-- 460cc83f02.pdb -> ../../cache/lines/...
|   |   |-- ced061f0b2.pdb -> ../../cache/lines/...
|   |   |-- d0cc72f231.pdb -> ../../cache/lines/...
|   |   `-- e51c741dc5.pdb -> ../../cache/lines/...
|   |-- makeblastdb_prot_each
|   |   `-- 138af179c1.pdb.list -> ../../cache/lines/...
|   |-- num
|   |   `-- a53b190835.num -> ../../cache/lines/...
|   |-- singletons
|   |   |-- 64733f5e6d.fna.list.list -> ../../cache/lines/...
|   |   `-- aaa2639d2e.faa.list.list -> ../../cache/lines/...
|   `-- str
|       |-- 1d8a61c770.str -> ../../cache/lines/...
|       |-- 4cdbcfa070.str -> ../../cache/lines/...
|       |-- 8fa8ede325.str -> ../../cache/lines/...
|       |-- a2fd369878.str -> ../../cache/lines/...
|       |-- cb201cf4f8.str -> ../../cache/lines/...
|       `-- db90f32127.str -> ../../cache/lines/...
`-- vars
    |-- mdbfaa.bht.list -> ../exprs/blastp_db_each/8158675169_1af4d085e7_5d8e23195d.bht.list
    |-- mdbfna.bht.list -> ../exprs/blastn_db_each/8158675169_bc4ea1fc06_992cadafb5.bht.list
    |-- mfafaa.bht.list -> ../exprs/blastp_each/8158675169_1af4d085e7_cd45a3683c.bht.list
    |-- mfafna.bht.list -> ../exprs/blastn_each/8158675169_bc4ea1fc06_43095607a3.bht.list
    |-- qfaa.faa -> ../exprs/load_faa/2d88028e40.faa
    |-- qfna.fna -> ../exprs/load_fna/82b637da24.fna
    |-- result -> ../exprs/any/3d347437fe.str.list
    |-- sdbfaa.bht.list -> ../exprs/list/487f4dec22.bht.list
    |-- sdbfna.bht.list -> ../exprs/list/0db8510f64.bht.list
    |-- sfaa1.faa -> ../exprs/load_faa/387f499ddc.faa
    |-- sfaa2.faa -> ../exprs/load_faa/e910806f83.faa
    |-- sfafaa.bht.list -> ../exprs/list/487f4dec22.bht.list
    |-- sfafna.bht.list -> ../exprs/list/0db8510f64.bht.list
    |-- sfna1.fna -> ../exprs/load_fna/eb12539f1c.fna
    `-- sfna2.fna -> ../exprs/load_fna/f110b4b1c9.fna

45 directories, 181 files
