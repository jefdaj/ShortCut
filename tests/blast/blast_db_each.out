["Synpcc7942_0003",
 "Synpcc7942_0006",
 "Synpcc7942_0010",
 "Synpcc7942_0010",
 "Synpcc7942_0005",
 "Synpcc7942_0004",
 "Synpcc7942_0009",
 "Synpcc7942_0008",
 "Synpcc7942_0001",
 "Synpcc7942_0003",
 "Synpcc7942_0007",
 "Synpcc7942_0008",
 "Synpcc7942_0002",
 "Synpcc7942_0001",
 "Synpcc7942_0005"]
